internal/cli/atlas/logs_download.go

```diff
@@ -114,7 +114,7 @@ func LogsDownloadBuilder() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.out, flag.Out, flag.OutShort, "", usage.LogOut)
+	cmd.Flags().StringVar(&opts.out, flag.Out, "", usage.LogOut)
```

Comment:
[q] I've spotted the removing `StringVarP` in a few places in this PR. I see in `flags.go` you've added `Output` and `OutputShort`. Any reason why you're not still reading both here?

Reply:
Ahh reading a bit more, this looks like you're keeping the legacy support for `Out` and adding the new flags `o` and `output`

Reply:
yeah the shorthand is now used for output which is global (and affects this command) and I can't have it be a short hand for this other flag :(